lib/active_merchant/billing/gateways/secure_pay_au.rb

```diff
@@ -61,6 +58,10 @@ def initialize(options = {})
         super
       end
 
+      def request_timeout
+        @options[:request_timeout] || 60
+      end
```

Comment:
Great. This should be really easy to push to upstream, right?

Reply:
Yep, will do! Who knows when a reviewer will get to it though.